.drone.yml

```diff
@@ -1,49 +1,36 @@
 pipeline:
   build_docker_image:
-    image: docker:1.12
+    image: docker:17.09.0-ce
     environment:
-      - DOCKER_HOST=tcp://127.0.0.1:2375
+      - DOCKER_HOST=tcp://172.17.0.1:2375
     commands:
-      - docker build -t image .
-
-  test:
```

Comment:
What happened with this step?

Reply:
Connection refused: https://drone.acp.homeoffice.gov.uk/UKHomeOffice/docker-nginx/50/4